src/main/java/com/robinpowered/sdk/model/Organization.java

```diff
@@ -17,11 +17,21 @@
 
 
     /**
+     * Properties
+     */
+
+    private UserAccess userAccess;
+
+
+    /**
      * Methods
      */
 
-    public Organization(int id, String slug, boolean isOrganization, DateTime createdAt, DateTime updatedAt) {
+    public Organization(int id, String slug, boolean isOrganization, DateTime createdAt, DateTime updatedAt,
+                        UserAccess userAccess) {
```

Comment:
In thinking about this, perhaps actually we make a `UserOrganization` entity that extends Organization and add the submodel there. THat may be the most "technically" correct representation of the API entities and wouldn't clutter the Org model with a submodel that is only relevant in a very specific context.